src/main/java/com/um/inventoryservice/PresentationLayer/OrderController.java

```diff
@@ -2,7 +2,6 @@
 
 import com.um.inventoryservice.BusinessLayer.OrderService;
 import com.um.inventoryservice.DataLayer.OrderDTO;
-import com.um.inventoryservice.DataLayer.StockItemDTO;
```

Comment:
A lot of StockItemDTO removals as well, just curious as to why.

Reply:
Not using them in the classes, must've been imported when copy/pasted some code.